packages/main/src/plugin/kubernetes-context-state.spec.ts

```diff
@@ -117,20 +123,20 @@ function fakeMakeInformer(
   }
   switch (path) {
     case '/api/v1/namespaces/ns1/pods':
-      return new FakeInformer(1, connectResult, [], []);
+      return new FakeInformer(kubeconfig.currentContext, path, 1, connectResult, [], []);
     case '/api/v1/namespaces/ns2/pods':
-      return new FakeInformer(2, connectResult, [], []);
+      return new FakeInformer(kubeconfig.currentContext, path, 2, connectResult, [], []);
     case '/api/v1/namespaces/default/pods':
-      return new FakeInformer(3, connectResult, [], []);
+      return new FakeInformer(kubeconfig.currentContext, path, 3, connectResult, [], []);
 
     case '/apis/apps/v1/namespaces/ns1/deployments':
-      return new FakeInformer(4, connectResult, [], []);
+      return new FakeInformer(kubeconfig.currentContext, path, 4, connectResult, [], []);
     case '/apis/apps/v1/namespaces/ns2/deployments':
-      return new FakeInformer(5, connectResult, [], []);
+      return new FakeInformer(kubeconfig.currentContext, path, 5, connectResult, [], []);
     case '/apis/apps/v1/namespaces/default/deployments':
-      return new FakeInformer(6, connectResult, [], []);
+      return new FakeInformer(kubeconfig.currentContext, path, 6, connectResult, [], []);
   }
-  return new FakeInformer(0, connectResult, [], []);
+  return new FakeInformer(kubeconfig.currentContext, path, 0, connectResult, [], []);
```

Comment:
looks like the only difference is the number so probably could do

```ts
case '/apis/apps/v1/namespaces/ns1/deployments':
   return buildFakeInformer(DEPLOYMENTS_NS1)
case '/apis/apps/v1/namespaces/ns2/deployments':
      return buildFakeInformer(DEPLOYMENTS_NS2)
```
and replace 4 by a constant like DEPLOYMENTS with value of 4

buildFakeInformer(id) -->
```
return new FakeInformer(kubeconfig.currentContext, path, id, connectResult, [], [])
```